docs/conf.py

```diff
@@ -124,7 +129,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#html_static_path = ['_static']
```

Comment:
I think I'd rather have an empty `_static/placeholder.txt`, to avoid future headscratching when adding (hypothetically) an image to illustrate something.